virt/about_virt/virt-security-policies.adoc

```diff
@@ -16,23 +16,26 @@ Learn about {VirtProductName} security and authorization.
 
 include::modules/virt-about-workload-security.adoc[leveloffset=+1]
 
-include::modules/virt-additional-scc-for-kubevirt-controller.adoc[leveloffset=+1]
+include::modules/virt-automatic-certificates-renewal.adoc[leveloffset=+1]
 
 [id="authorization_virt-security-policies"]
 == Authorization
```

Comment:
A nitpick, but for searchability and findability, you might consider adding some info to this heading and the `TLS certificates` heading so that they are not so generic and instead include specific references to virtualization.